codebreaker.py

```diff
@@ -1,34 +1,75 @@
-trueNumber = "1010";
-
-class Codebreaker:
-    
-  def adivinar(self, numero=None):
-    if trueNumber == '':
-      return 'Number is not defined'
-
-    if numero is None or len(numero) != 4 or 'e' not in list(numero):
-      return "error"
-    
-    if numero == trueNumber:
-      return True
-
-    resultadoX  = ''
-    resultado_  = ''
-    arrayNumber = []
-    
-    for x in len(numero):
-      if(arrayNumber[numero[x]] == True):
-        return 'error'
-
-      arrayNumber[numero[x]] = True
-    
-    numero = list(numero)
-
-    for index, x in numero:
-      if trueNumber[index] == numero[index]:
-        resultadoX+='X'
-
-      elif x in trueNumber:
-        resultado_='_'
-
-    return resultadoX+resultado_
+""" Class file for CodeBreaker"""
+# __doc__ (Class for little game CodeBreaker where you must guess
+# the secret number)
+
+SECRET_NUMBER = '1010'  # SECRET NUMBER CONSTANT
```

Comment:
Remember the original number should not have duplicated digits.

Reply:
True, I forgot to update the code here in this fork, but I uploaded a month ago here https://github.com/gsmx64/python-adv-bootcamp-cf/commit/36d824213a1854fd54c5e345c3e6615c2fd9eeca#diff-498e04ded81b2d4e792e100c899be5f1844e37bd70dfb21e4b760543b7f9ef0c